openfisca_us/variables/irs/tax/payroll/payroll_tax_gross_wages.py

```diff
@@ -0,0 +1,12 @@
+from openfisca_us.model_api import *
+
+
+class payroll_tax_gross_wages(Variable):
+    value_type = float
+    entity = Person
+    label = "Gross wages and salaries for payroll taxes"
+    definition_period = YEAR
+    unit = USD
+
+    def formula(person, period):
```

Comment:
By the way, the pattern in OpenFisca-Tools I made a while ago allows:

```python
  unit = USD

  formula = sum_of_variables([var1, var2, ...])
```

[It'll also work with a list parameter name instead of a list of variables](https://github.com/PolicyEngine/openfisca-tools/blob/master/openfisca_tools/model_api.py)